### PR TITLE
Improved meta in country pages

### PR DESCRIPTION
**Pull request by kronaemmanuel**
Helps with #202 

- Changed the og:description to show a summary of the measurements about the country
- Changed the og:title to show the appropriate title with current country's name in it

Possible improvements: use Intl strings instead of plain english

### CODE REVIEW

File: components/Header.js [hunk 1]
```diff
@@ -26,10 +26,9 @@ export default class Header extends React.Component {
           <meta name="twitter:site" content="@openobservatory" />
           <meta name="twitter:creator" content="@openobservatory" />
 
-          <meta name='og:title' property='og:title' content='OONI Explorer - Open Data on Internet Censorship Worldwide' />
-          <meta property="og:description" content={description} />
+          <meta key="og:title" name='og:title' property='og:title' content='OONI Explorer - Open Data on Internet Censorship Worldwide' />
```
[the remark this answers is not in the document]
Comment (kronaemmanuel) [reply components/Header.js]: We have a default ```og:description``` in Header.js and then a custom description in Country.js.
If we don't use ```key```, then Next will give us both descriptions. ```key``` prevents duplication and only gives the latter meta tag which in this case is the one in Country.js.

File: components/Header.js [hunk 2]
```diff
@@ -26,10 +26,9 @@ export default class Header extends React.Component {
...
+          <meta  key="og:description" property="og:description" content={description} />
```
[the remark this answers is not in the document]
Comment (kronaemmanuel) [reply components/Header.js]: Ahhhh, I'm working on a version of this that uses React Intl, so this will get solved in that. :sweat_smile:

**Comment by kronaemmanuel** [comment]
Hey @sarathms, I seem to have messed up the yarn.lock file and I don't know how to resolve this conflict. I think i accidentally upgraded some packages.

Other than that: I have implemented the meta tags with Intl package.

File: components/Header.js [hunk 3]
```diff
@@ -26,8 +26,8 @@ export default class Header extends React.Component {
           <meta name="twitter:site" content="@openobservatory" />
           <meta name="twitter:creator" content="@openobservatory" />
 
-          <meta name='og:title' property='og:title' content='OONI Explorer - Open Data on Internet Censorship Worldwide' />
-          <meta property="og:description" content={description} />
+          <meta key="og:title" name='og:title' property='og:title' content='OONI Explorer - Open Data on Internet Censorship Worldwide' />
+          <meta key="og:description" property="og:description" content={description} />
```
Comment (sarathms) [inline components/Header.js:30]: I think there is inconsistency (even before this PR) in how we use `property` and `name` attributes. But I have opened another issue (#450) to check that. We can retain this as it is for now.

File: pages/country.js [hunk 4]
```diff
@@ -43,6 +44,49 @@ const AnimatedFlex = styled(Flex)`
 const AnimatedHeading = styled(Heading)`
   transition: all 0.5s ease;
 `
+const IntlHead = ({
```
Comment (sarathms) [inline pages/country.js:47]: Let's move this component into its own file under `components/country/`

File: pages/country.js [hunk 5]
```diff
@@ -100,17 +144,14 @@ export default class Country extends React.Component {
       countryCode,
       countryName,
       overviewStats,
-      reports
+      reports,
```
Comment (sarathms) [inline pages/country.js:147]: can remove this

File: components/Header.js (hunk 3 above)
Comment (kronaemmanuel) [reply components/Header.js:30]: Ok, I'll look that up.

File: pages/country.js (hunk 4 above)
Comment (kronaemmanuel) [reply pages/country.js:47]: Ok, I'll do that and send another PR.

File: pages/country.js (hunk 5 above)
Comment (kronaemmanuel) [reply pages/country.js:147]: Sure.

**Comment by kronaemmanuel** [comment]
The issues you pointed out have been solved but I ran ```yarn``` and now I have a conflicting ```yarn.lock``` file. Can you take a look at why that would be the case?

**Comment by kronaemmanuel** [comment]
@sarathms, I've reset the ```yarn.lock``` file. This can be merged now :smile:

**Pull request merged**